src/material-experimental/mdc-snack-bar/_snack-bar-theme.scss

```diff
@@ -36,7 +36,7 @@
   mdc-snackbar.$dismiss-ink-color: $orig-dismiss-ink-color;
 
   // The extra level of nesting here is to increase the specificity.
-  .mat-mdc-simple-snack-bar .mat-mdc-snack-bar-actions .mdc-snackbar__action {
+  .mat-mdc-simple-snack-bar .mat-mdc-snack-bar-actions .mdc-snackbar__action.mdc-snackbar__action {
```

Comment:
Might be worth adding a comment why it needs the selector. Also maybe we can change it to something like `button.mdc-snackbar__action` so it looks less weird.

Reply:
I expanded the existing comment to give a little more detail. I don't think we can use `button.mdc-snackbar__action` because its not guaranteed to be a button